patroni/postgresql.py

```diff
@@ -78,6 +78,7 @@ def __init__(self, config):
         self.replication_slots = []  # list of already existing replication slots
         self.retry = Retry(max_tries=-1, deadline=5, max_delay=1, retry_exceptions=PostgresConnectionException)
 
+        self._creating_replica = False
```

Comment:
Ok, I've found a better solution.
We should use `Postgresql.state` property for that. It is already exposed via REST API.
And regarding `sysid == 0`: if sysid is not defined  it is not exposed.